tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/RestSharp/UrlEncodeIntegration.cs

```diff
@@ -0,0 +1,81 @@
+// <copyright file="UrlEncodeIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Iast;
+using Datadog.Trace.Vendors.Serilog;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.RestSharp;
+
+/// <summary>
+/// System.Security.Cryptography.HashAlgorithm instrumentation
+/// </summary>
+[InstrumentMethod(
+    AssemblyName = "RestSharp",
+    TypeName = "RestSharp.Extensions.StringExtensions",
+    MethodName = "UrlEncode",
+    ReturnTypeName = ClrNames.String,
+    ParameterTypeNames = new[] { ClrNames.String },
+    MinimumVersion = "1.0.0",
+    MaximumVersion = "112.*.*",
```

Comment:
Uh is this correct that we support v 1.0.0 up to 112 with this?

Reply:
True. first version is 104.0.0